src/setup/rgw.py

```diff
@@ -88,6 +89,16 @@ def config(conf: MappedConf):
 
         Log.info('Config phase started.')
 
+        Log.info("create symbolic link of FID config files started")
+        Rgw._create_symbolic_link_fid(conf)
+        Log.info("create symbolic link of FID config files completed")
+        Log.info("fetching endpoint values from hare sysconfig file.")
+        # for running radosgw-admin tool, we are using endpoints mentioned
```

Comment:
Need to have more details in this comment, update on following lines.

For running rgw service and radosgw-admin tool, using same endpoints mentioned in first symlink file 'rgw-1' as default endpoints, given radosgw-admin tool & rgw service not expected to run simultaneously. Radosgw-admin is used only in mini provisioner phase i.e in Init container, and then later rgw service container boots up.

Reply:
done